src/components/scenes/DuressPinScene.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import { Keyboard, ScrollView } from 'react-native'
+import { cacheStyles } from 'react-native-patina'
+import { makeEvent } from 'yavent'
+
+import { useHandler } from '../../hooks/useHandler'
+import { useScrollToEnd } from '../../hooks/useScrollToEnd'
+import { lstrings } from '../../locales/strings'
+import { EdgeAppSceneProps } from '../../types/routerTypes'
+import { logActivity } from '../../util/logger'
+import { SceneButtons } from '../buttons/SceneButtons'
+import { EdgeAnim } from '../common/EdgeAnim'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { SceneContainer } from '../layout/SceneContainer'
+import { Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+import { SceneHeaderUi4 } from '../themed/SceneHeaderUi4'
+import { DigitInput, MAX_PIN_LENGTH } from './inputs/DigitInput'
+
+interface Props extends EdgeAppSceneProps<'duressPin'> {}
+
+export const DuressPinScene = (props: Props) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const [pin, setPin] = React.useState('')
+  const isValidPin = pin.length === MAX_PIN_LENGTH
+
+  const handleComplete = () => {
+    if (!isValidPin) return
+    logActivity(`Duress PIN selected`)
+    emitPin(pin)
+  }
+
+  const handleChangePin = useHandler((newPin: string) => {
+    // Change pin only when input are numbers
+    if ((/^\d+$/.test(newPin) || newPin.length === 0) && newPin.length <= 4) {
+      setPin(newPin)
+      if (newPin.length === 4) {
+        Keyboard.dismiss()
+      }
+    }
+  })
+
+  const scrollViewRef = useScrollToEnd(isValidPin)
+
+  return (
+    <SceneWrapper scroll={false}>
+      <SceneContainer expand>
+        <SceneHeaderUi4 title={lstrings.title_set_duress_pin} />
+        <ScrollView ref={scrollViewRef} contentContainerStyle={styles.content} keyboardShouldPersistTaps="handled">
```

Comment:
See relevant comment in `DuressModeSettingScene`

Reply:
See which comment for what?

Reply:
There's only one comment in that file